src/client/lib/query.js

```diff
@@ -43,6 +44,7 @@ module.exports = class Query {
         gzip: true
       };
 
+      debug(`http request: ${JSON.stringify(this.options)}`);
```

Comment:
Also would be nice to name the method logging. Something like : `debug('run => http://google.com?thequey=segsdfg')`